app/views/nic/_base_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= selectable_f f, :type, interfaces_types, {},
-                 :class => 'interface_type', :disabled => !f.object.new_record? %>
+                 :class => 'interface_type without_select2', :disabled => !f.object.new_record? %>
```

Comment:
@domcleal This seems to work for me, what are you doing exactly that doesn't work (i.e. new/existing host/interface)? are there any errors in the js console? did you try clearing the cache (maybe you still use previous js version)?